### ISSUE TITLE
BLT not working

### ISSUE THREAD

**Issue by TheTakkle** [JamesWilko/Payday-2-BLT#166]
My BLT is not working. When i start up Payday 2, its like a vanilla Payday 2. No BLT mods updater button thingy on the main menu.
I've tried verifying the game cache, reinstalling the latest version of BLT (r7_r19) over and over again.

**Comment by PJzuza** [comment]
Like I said. Provide all information that the dev should know
- Your PC spec. 
- Your Window version.

**Comment by TheTakkle** [comment]
My PC specs:
Intel Core i3-6100
8 GB RAM
RX 460

My windows version:
Version 1709 (OS Build 16299. 19)

**Comment by PJzuza** [comment]
@TheTakkle ~~can you run the game if you delete your mods folder and dll file? If not then it may be the base game problem but I'm not sure but if you can then there is something with the BLT.~~ ah nevermind you can but BLT is not working my bad

Anyway I forgot to suggest that you should reinstall the x86 2013 C++ Runtime  again using this link : http://download.microsoft.com/download/2/E/6/2E61CFA4-993B-4DD4-91DA-3737CD5CD6E3/vcredist_x86.exe?ranMID=24542&ranEAID=je6NUbpObpQ&ranSiteID=je6NUbpObpQ-p1fCdDM8eRS6_ZLTJrzgpw and try your game with BLT again.

**Comment by TheTakkle** [comment]
@PJzuza Yes I can run it perfectly fine.

**Comment by PJzuza** [comment]
so you can run the game together with BLT now? or just vanilla game? if your problem has been solved then please close the thread

**Comment by TheTakkle** [comment]
It's just the vanilla game. Even with BLT folders.

**Comment by TheTakkle** [comment]
Reinstalled the x86 2013 C++ and it's still not working.

**Comment by AlphaDelta9** [comment]
me too after some update i play again my payday 2 and BLT not working now

**Comment by dog194** [comment]
Try to delete this file     payday2/mods/save/blt_data.txt 
then Restart the game.

**Comment by TryxxStyxx6** [comment]
Please make sure you have BLT 2.3, which is the most current version, which can be found here:

http://paydaymods.com/download/

Install same as before of course, then let us know if that helps :)

**Comment by TryxxStyxx6** [comment]
Please make sure you have BLT 2.3, which is the most current version, which can be found here:

http://paydaymods.com/blog/13/luajit-update/

Install same as before of course, then let us know if that helps :)

**Comment by TheTakkle** [comment]
@dog194 There's nothing called "blt_data.txt" inside the saves folder.

**Comment by TheTakkle** [comment]
@TryxxStyxx6 I already downloaded the latest version. I re-downloaded the BLT, and installed it correctly. Still not working.

**Comment by TryxxStyxx6** [comment]
If the mods folder AND the IPHLPAPI.dll file are not both in the steam/steamapps/common/PAYDAY 2 Folder, then it will not work. Please double check that's the case. I know that is a really simple process, but a lot of people seem to slip up there haha

**Comment by TheTakkle** [comment]
@TryxxStyxx6 Yes, both of them are there.

**Comment by TryxxStyxx6** [comment]
Honestly I'm not sure. It should have been as simple as moving the mods folder and the IPHLPAPI.dll into your PAYDAY2 folder.. Could you send a screen shot of the payday 2 folder and inside the mods folder? Maybe there's a file missing. Also you should have that blt_data.txt so that might be the issue

**Comment by TheTakkle** [comment]
![payday2folder](https://user-images.githubusercontent.com/33158706/32169881-cdd5fca0-bda4-11e7-8aed-c51081a74e9b.JPG)
![modsfolder](https://user-images.githubusercontent.com/33158706/32169883-ce914618-bda4-11e7-9875-daa769d9f96d.JPG)

**Comment by TheTakkle** [comment]
@TryxxStyxx6 That's weird. Because there was never any file named "blt_data.txt" in the saves folder, even in the BLT zip file. There's only 2 txt files named "blt_revision", and "saves".

**Comment by TryxxStyxx6** [comment]
Here this is the blt_data.txt file that I have. Try putting it in your save folder and see if it works
[
[blt_data.txt](https://github.com/JamesWilko/Payday-2-BLT/files/1428600/blt_data.txt)
](url)

**Comment by TheMTNDewd** [comment]
I Have the exact same issue. BLT is installed correctly, no other mods (I did have Microsoft Jacket, but I even removed it just in case) and BLT just stopped working all of a sudden last week.

**Comment by dmr8888** [comment]
I fixed the problem by running the game's .exe directly instead of running from the Steam client.

**Comment by IAmJacket** [comment]
@TheTakkle Did you ever get a solution to this? Im running into the same problem.

**Comment by TheTakkle** [comment]
Well. As it turns out I only have to turn off compatibility mode. And I just figured it out recently.

**Comment by Timooooooh** [comment]
@TheTakkle where can you do that?

**Comment by AlphaDelta9** [comment]
Right-Click payday2_win32_release, select Properties, Select Compatibility, Uncheck Disable Fullscreen Optimization

**Comment by OmarAminAI** [comment]
after the last update my BLT isnt working anymore need some help over here!

**Comment by Knakendraaier** [comment]
Still not working.

**Comment by GaryTh3Goat** [comment]
the solution from @TryxxStyxx6 worked with the save file GENIUS 200 IQ <3

**Comment by oofing** [comment]
Microsoft is quite rude. Here is my solution, it worked for me! Go to your: payday2_win32_release.exe, Right click it, click properties, Select compatibility, press change settings for all users, and if checked, un-check Run this in compatibility mode for: 
Backstory of how this problem probably originated (this was how it originated for me) : Payday 2 wasn't opening properly, so I opened the compatibility settings, and ran the troubleshooter, I didn't know the issue so I chose I don't know for everything, after that, I opened my payday 2, and poof my game was vanilla. 
Tip: Don't use troubleshooters without knowing what it does to your game. 
If it worked, I'm glad I could help. 
Happy modding.

**Comment by NoName3034** [comment]
Hi guys i have found a 100% working fix here is a new BLT loader Try SuperBLT superblt.znix.xyz read what thay say befor you copy files
